app/src/main/res/values/strings.xml

```diff
@@ -186,7 +186,7 @@
     <string name="journal_item_attendees">Attendees</string>
     <string name="journal_item_reminders">Reminders</string>
     <string name="journal_item_phone">Phone</string>
-    <string name="journal_item_email">Email</string>
+    <string name="journal_item_email">E-mail</string>
```

Comment:
I think Email is better

Reply:
It fails in translation, where variants that don't have the dash are not allowed in other languages. It is an inconsistency found pretty uniquely in English.

Reply:
Not true. In French, for example, electronic email is translated to courrier électronique, of which the short version is courriel. Though even if it's a unique English thing, that's how it's common in English. I don't think it makes translations harder as people who translate know what it means and can translate it as they see fit.

Reply:
Not a problem in French. You can bait people into making a mistake there with "mail".
In EteSync is already wrong in Dutch, Italian, Hungarian, Czech, Spanish and Turkish. (Possibly others.) With more translations to come, and "*-*" being the "proper" format even in English, why not?

https://hosted.weblate.org/translate/etesync/android/nb_NO/?checksum=c421798ea9d8d8aa
https://hosted.weblate.org/translate/etesync/android/nb_NO/?checksum=c698a7d2e3cad176
https://hosted.weblate.org/translate/etesync/android/nb_NO/?checksum=c421798ea9d8d8aa

I have never seen it not produce errors in translation.

Reply:
What do you mean by produce errors in translation? I really don't understand how E-mail will make it better than Email for translators.
As for it being valid: valid doesn't necessarily imply common. It's similar to the long form of email: "electronic mail". Any user that would see that would find it odd at best. I believe E-mail falls into the same category (though less bad).

Reply:
Likely results for translators of languages that only allow ~"e-word":
Email: Knows the word, forgets to put in hyphen, or thinks it is a stylistic choice on part of the app.
E-mail: Knows the word, remembers the hyphen.

Likely results for translators of languages that only allow ~"eword":
Email: Knows the word, translates the word
E-mail: Knows the word, doesn't put in hyphen by mistake because it is a far less likely error to ever occur.

In addition to this, "e-mail" is considered to be more proper, and was only recently allowed in the Chicago style-guide because they couldn't afford to hire people to weed it out in their publications.

While you have a bit of a point, it is outweighed. Prescriptive and descriptive grammar, etc. I think for English, it will boil down to "mail" in the future, and what is now mail will be "snail mail" in English. And at last a "mail server" will send "mail".